lib/alerts/parser.ex

```diff
@@ -103,6 +103,7 @@ defmodule Alerts.Parser do
   defp cause("UNKNOWN_CAUSE"), do: :unknown_cause
   defp cause("UNRULY_PASSENGER"), do: :unruly_passenger
   defp cause("WEATHER"), do: :weather
+  defp cause(_), do: :other_cause
```

Comment:
As a discussion for later, maybe there's no point to doing all this matching and we should just parse the cause as an atom whatever it is.

Reply:
Yep, I completely agree. I think this data is much less standardized than I originally thought it was.

Reply:
[Ticket for the follow-up improvement](https://app.asana.com/0/1167196461144361/1206230599218673)